tests/entrypoint/src/main.rs

```diff
@@ -0,0 +1,36 @@
+#![no_main]
+athena_vm::entrypoint!(main);
+
+use athena_vm::helpers::address_to_32bit_words;
+use athena_vm::types::ADDRESS_ALICE;
+use athena_vm_declare::{callable, template};
+use athena_vm_sdk::call;
+
+// Note: the test harness installs this contract code at ADDRESS_ALICE
+
+pub fn main() {
+  // recurse forever
+  let address = address_to_32bit_words(ADDRESS_ALICE);
+  let value: [u32; 2] = [0, 0];
+  unsafe { athena_vm::host::call(address.as_ptr(), std::ptr::null(), 0, value.as_ptr()) };
+}
```

Comment:
How about passing the address to call in the input rather than assuming that Host installed this contract at specific address? Something like:
```suggestion
pub fn main() {
  let input = athena_vm::io::read_vec::<Address>();
  let address = bincode::deserialize(&input).unwrap();
  athena_vm_sdk::call(address, Some(input), 0);
}
```